engine/vcs/vcs.go

```diff
@@ -107,7 +107,7 @@ func (s *Service) getConsumer(name string) (sdk.VCSServer, error) {
 		return gitlab.New(serverCfg.Gitlab.AppID,
 			serverCfg.Gitlab.Secret,
 			serverCfg.URL,
-			s.Cfg.API.HTTP.URL+"/repositories_manager/oauth2/callback",
```

Comment:
Why not keep this value as default if we don't indicate `OAuthCallbackURL` ?

Reply:
Because the other value doesn't have any sens :/